### DISCUSSION TITLE
LDM Notes for September 26th, 2022

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#6495] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2022/LDM-2022-09-26.md

## Agenda

- Working set triage
    - Roles & Extensions
    - Discriminated Unions
    - Bridging statements and expressions
    - Construction improvements
    - `ref struct` improvements
    - `params` improvements
    - Ungrouped

**Comment by huoyaoyuan** [comment]
> **Roles & Extensions**
> We already have an internal working group for this topic, and they will be continuing to work on it.

Glad to hear that, and thanks for sharing your status. I'm expecting your design ideas when you are ready.

> **Bridging statements and expressions**

This is another thing that I value more.

**Comment by Richiban** [comment]
As someone who's been trying to get discriminated unions into C# for almost a decade, I'm very, very happy that you're looking at them now 😊
